### PR TITLE
Feature/steps volatility

### PR DESCRIPTION
**Pull request by ChinW97**

### CODE REVIEW

File: .pytest_cache/v/cache/nodeids [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (Meng6) [inline .pytest_cache/v/cache/nodeids:1]: Why do we add this file? Can we delete it?

File: tests/settings/mtz_event_config.yaml [hunk 2]
```diff
@@ -50,13 +50,13 @@ PHONE_DATA_STREAMS:
   
   # AVAILABLE:
   aware_mysql: 
-    DATABASE_GROUP: MY_GROUP
+    DATABASE_GROUP: RHYTHMS
```
Comment (Meng6) [inline tests/settings/mtz_event_config.yaml:53]: Change all the `DATABASE_GROUP` fields back to `MY_GROUP`, please.

**Review by Meng6 — CHANGES_REQUESTED** [review]
Looks good, @ChinW97 and @JimZhu96 ! Besides the above comments, could you please:
- Update the `docs/change-log.md` file
- Add descriptions for these steps features: `docs/features/fitbit-steps-summary.md`

File: tests/settings/mtz_event_config.yaml (hunk 2 above)
Comment (ChinW97) [reply tests/settings/mtz_event_config.yaml:53]: Changed!

File: .pytest_cache/v/cache/nodeids (hunk 1 above)
Comment (ChinW97) [reply .pytest_cache/v/cache/nodeids]: Deleted

File: .pytest_cache/v/cache/stepwise [hunk 3]
```diff
@@ -0,0 +1 @@
+[]
```
[the remark this answers is not in the document]
Comment (ChinW97) [reply .pytest_cache/v/cache/stepwise]: Deleted

**Pull request merged**